src/shared/hooks/use-linked-interactives-authoring.test.ts

```diff
@@ -108,7 +108,7 @@ describe("useLinkedInteractives", () => {
         schema: {}
       });
     };
-    renderHook(HookWrapper);
+    act(() => { renderHook(HookWrapper); });
```

Comment:
In my experience, `act()` is normally used for things you do to a component _after_ it's rendered, e.g.
```
  renderHook(...);
  act(() => userEvent.click());
```
Was there a particularly reason you needed to wrap the renders themselves in `act()`?